sphinx/jinja2glue.py

```diff
@@ -208,13 +211,13 @@ def get_source(self, environment: Environment, template: str) -> tuple[str, str,
         loaders = self.loaders
         # exclamation mark starts search from theme
         if template.startswith('!'):
-            loaders = loaders[self.templatepathlen:]
+            loaders = loaders[self.templatepathlen :]
             template = template[1:]
         for loader in loaders:
             try:
                 return loader.get_source(environment, template)
             except TemplateNotFound:
                 pass
-        msg = (f"{template!r} not found in "
-               f"{self.environment.loader.pathchain}")  # type: ignore[union-attr]
+        pathchain = f'{self.environment.loader.pathchain}'  # type: ignore[union-attr]
```

Comment:
This f-string is useless.

Reply:
i suggest suppressing the flak8 lint

Reply:
> i suggest suppressing the flak8 lint

Ok I just increased the flake8 line-length to 120, and then put all this back on one line